docs/data/data-core-editor.md

```diff
@@ -1812,6 +1842,7 @@ only relevant when the annotation is not a block annotation.
 
  * annotation: The annotation to add.
  * blockClientId: The blockClientId to add the annotation to.
+ * blockAttribute: The attribute of the block to add the annotation to.
```

Comment:
Would this be probably more clear if it's named "richTextIdentifier or something similar? as if I understand properly it's not always 1 <=> 1 with attributes.

Reply:
Yes, absolutely. With simple blocks it will probably be the attribute, but with more complex blocks it won't.